tests/vmipreset_test.go

```diff
@@ -300,15 +303,22 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	})
 
 	Context("Conflict", func() {
-		var conflictPreset *v1.VirtualMachineInstancePreset
+		const (
+			conflictFlavor = "conflict-test"
+			conflictPrefix = "test-conflict-"
+		)
 
-		conflictKey := fmt.Sprintf("%s/conflict", core.GroupName)
-		conflictFlavor := "conflict-test"
-		conflictMemory, _ := resource.ParseQuantity("256M")
-		conflictPrefix := "test-conflict-"
+		var (
+			conflictPreset *v1.VirtualMachineInstancePreset
+			conflictKey    string
+			conflictMemory resource.Quantity
+		)
 
 		BeforeEach(func() {
+			conflictKey = fmt.Sprintf("%s/conflict", core.GroupName)
```

Comment:
nit: I think this one can be set outside before-each `conflictKey = core.GroupName + "/conflict"`

Reply:
@ormergi - fixed. PTAL.